src/Serilog/ILogger.cs

```diff
@@ -1452,16 +1327,14 @@ void Fatal(Exception? exception, string messageTemplate, params object?[]? prope
         /// render the <paramref name="boundProperties"/> as text.</param>
         /// <param name="boundProperties">Captured properties from the template and <paramref name="propertyValues"/>.</param>
         /// <example>
-        /// <code>
         /// MessageTemplate template;
-        /// IEnumerable&lt;LogEventProperty&gt; properties;
+        /// IEnumerable&lt;LogEventProperty&gt; properties>;
```

Comment:
this line does not look right (have not checked rest in detail)

Reply:
Nice spotting; pushed the fix to https://github.com/serilog/serilog/pull/1669 (this reimplements #1664, which had to be reverted)